src/applications/claims-status/containers/OverviewPage.jsx

```diff
@@ -188,13 +192,22 @@ class OverviewPage extends React.Component {
     }
 
     const { claimPhaseDates } = claim.attributes;
+    const currentPhase = getPhaseFromStatus(claimPhaseDates.latestPhaseType);
 
     return (
       <div className="overview-container">
         <ClaimOverviewHeader />
+        <Toggler toggleName={Toggler.TOGGLE_NAMES.cstClaimPhases}>
+          <Toggler.Enabled>
+            <div className="claim-phase-diagram">
+              <MobileClaimPhaseDiagram currentPhase={currentPhase} />
+              <DesktopClaimPhaseDiagram currentPhase={currentPhase} />
+            </div>
+          </Toggler.Enabled>
+        </Toggler>
         <ClaimTimeline
```

Comment:
The `ClaimTimeline` component should be hidden when the Phase diagrams are visible

Reply:
Since that is replaced by the accordian I didnt think it needed to be a part of this ticket. I plan on implementing that  when I work these tickets
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/82370
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/82377
 I didnt think it was necessary to do that yet.

Reply:
I just saw this acceptance criteria on the ticket and wanted to make sure it was accounted for:
<img width="472" alt="Screenshot 2024-05-10 at 1 59 01 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/13838621/78ec5963-72a0-4c2d-a743-1216c871d4eb">. If product is fine with changing the acceptance criteria and moving it to another ticket then all is good

Reply:
Ill move that ac to the tickets I mentioned, I dont think it makes sense to do that work yet!

Reply:
@jerekshoe removed that ac and updated https://github.com/department-of-veterans-affairs/va.gov-team/issues/82370 with a task for that and a/c